packages/wordcount/src/index.js

```diff
@@ -0,0 +1,102 @@
+import { extend, flow  } from 'lodash';
+import { defaultSettings } from './defaultSettings'
+import stripTags from './stripTags';
+import transposeAstralsToCountableChar from './transposeAstralsToCountableChar';
+import stripHTMLEntities from './stripHTMLEntities';
+import stripConnectors from './stripConnectors';
+import stripRemovables from './stripRemovables';
+import stripHTMLComments from './stripHTMLComments';
+import stripShortcodes from './stripShortcodes';
+import stripSpaces from './stripSpaces';
+import transposeHTMLEntitiesToCountableChars from './transposeHTMLEntitiesToCountableChars';
+
+
+
+/**
+ * Private function to manage the settings.
+ *
+ * @param type
+ * @param userSettings
+ * @returns {void|Object|*}
+ */
```

Comment:
I believe the last recommendation, we use `@return` instead of return. And we should type each param and add a description. Something like

```
* @param {string}           type              short description of the type arg
* @param {Object}         userSettings short description of the user settings 

* @returns {void|Object|*} description of the returned value
```